src/backend/marsha/core/views.py

```diff
@@ -786,8 +786,16 @@ def _get_app_data(self, lti, playlist):
             context={"request": self.request},
         ).data
 
+        all_videos = get_selectable_resources(Video, lti)
+
         videos = VideoSelectLTISerializer(
-            get_selectable_resources(Video, lti),
+            all_videos.filter(live_type__isnull=True),
+            many=True,
+            context={"request": self.request},
+        ).data
+
+        webinars = VideoSelectLTISerializer(
+            all_videos.filter(live_type__isnull=False),
```

Comment:
Just curious, how do you add bbb meetings? Should we soon think of splitting everything about video/live into a separate app (called something like "interactive"?) and keep only shared services in the `core` app?

Reply:
Meetings (and Markdown Documents in his own app) are loaded through API https://github.com/openfun/marsha/blob/master/src/backend/marsha/bbb/api.py#L72
This is the way others resources should be added.

Reply:
`and live_state!=ended`